### ISSUE TITLE
Yahoo 403 Client Error: Forbidden URL

### ISSUE THREAD

**Issue by Vendrel** [kaito1410/AutoDD_Rev2#45]
Kaito, you are the patron saint of day-traders and swing-traders!

It seems, Yahoo pushed our nose to the glass:
![image](https://user-images.githubusercontent.com/5756032/127541007-dec7daa6-c589-4fa1-9507-65ce2b98897b.png)

I hope you can resolve it. 🤘🏼

**Comment by mxchinegod** [comment]
@Vendrel Hello, I fixed this in my fork.

https://github.com/DylanAlloy/AutoDD_Rev2

You can also install mine with pip: `pip install autodd && autodd` :)

**Comment by cokeito** [comment]
@DylanAlloy , i tried your fork and it's not working as today(23-10-03)

401 Client Error: Unauthorized for url: https://query2.finance.yahoo.com/v7/finance/quote?formatted=True&symbols=CNXA%2CRSI%2CNVOS%2CMACD%2CSPY%2CQQQ%2CSPX%2CATR%2CISM%2CPTSD%2CONLY%2CJPOW%2CJOLTS%2CVIX%2CWEED%2CVWAP%2CWSJ%2CABT%2CTGT%2CDOW%2CNKE%2CIVZ%2CBTI%2CMMM%2CGIS%2CPFE%2CRTX%2CBAC%2CCPB%2CSMG%2CIRA%2CVOO%2CTLT%2CIBKR%2CFAQ%2CDCA%2CSNSXX%2CHYSA%2CKBW%2CRBC%2CCFRA%2CFDIC%2CLAC%2CUAW%2CRULES%2CHOUSE%2CSVB%2CRIP%2CTSLA%2CCVNA%2CUNH%2CUBS%2CMSOX%2COPEC%2CRKLB%2CNDA%2CAMZN%2CULA%2CCBDW%2CNFTG%2COLB%2CLRE%2CTLRY%2CPST%2CSIRI%2CVNQ%2CNET%2CWHAT%2CLTD%2CSCE%2CQQM%2CLOT%2CNEE%2CVIPSX%2CVMFXX%2CYTD%2CAPR%2CCIT%2CPER%2CATM%2CPBR%2CDMA%2CBOA%2CDOES%2CSOME%2CCDC%2CIHS%2CRIG&fields=regularMarketPreviousClose%2CfiftyDayAverage%2CregularMarketVolume%2CaverageDailyVolume3Month%2CregularMarketPrice%2CregularMarketChangePercent%2CfloatShares

can you take a look on it?
thanks in advance.

**Comment by DylanAlloy** [comment]
> @DylanAlloy , i tried your fork and it's not working as today(23-10-03)

> 401 Client Error: Unauthorized for url: https://query2.finance.yahoo.com/v7/finance/quote?formatted=True&symbols=CNXA%2CRSI%2CNVOS%2CMACD%2CSPY%2CQQQ%2CSPX%2CATR%2CISM%2CPTSD%2CONLY%2CJPOW%2CJOLTS%2CVIX%2CWEED%2CVWAP%2CWSJ%2CABT%2CTGT%2CDOW%2CNKE%2CIVZ%2CBTI%2CMMM%2CGIS%2CPFE%2CRTX%2CBAC%2CCPB%2CSMG%2CIRA%2CVOO%2CTLT%2CIBKR%2CFAQ%2CDCA%2CSNSXX%2CHYSA%2CKBW%2CRBC%2CCFRA%2CFDIC%2CLAC%2CUAW%2CRULES%2CHOUSE%2CSVB%2CRIP%2CTSLA%2CCVNA%2CUNH%2CUBS%2CMSOX%2COPEC%2CRKLB%2CNDA%2CAMZN%2CULA%2CCBDW%2CNFTG%2COLB%2CLRE%2CTLRY%2CPST%2CSIRI%2CVNQ%2CNET%2CWHAT%2CLTD%2CSCE%2CQQM%2CLOT%2CNEE%2CVIPSX%2CVMFXX%2CYTD%2CAPR%2CCIT%2CPER%2CATM%2CPBR%2CDMA%2CBOA%2CDOES%2CSOME%2CCDC%2CIHS%2CRIG&fields=regularMarketPreviousClose%2CfiftyDayAverage%2CregularMarketVolume%2CaverageDailyVolume3Month%2CregularMarketPrice%2CregularMarketChangePercent%2CfloatShares

> can you take a look on it? thanks in advance.

Yes to make sure I keep track of this would you mind making an issue on https://github.com/DylanAlloy/AutoDD_Rev2

**Comment by mxchinegod** [comment]
@cokeito 

> @DylanAlloy , i tried your fork and it's not working as today(23-10-03)

> 401 Client Error: Unauthorized for url: https://query2.finance.yahoo.com/v7/finance/quote?formatted=True&symbols=CNXA%2CRSI%2CNVOS%2CMACD%2CSPY%2CQQQ%2CSPX%2CATR%2CISM%2CPTSD%2CONLY%2CJPOW%2CJOLTS%2CVIX%2CWEED%2CVWAP%2CWSJ%2CABT%2CTGT%2CDOW%2CNKE%2CIVZ%2CBTI%2CMMM%2CGIS%2CPFE%2CRTX%2CBAC%2CCPB%2CSMG%2CIRA%2CVOO%2CTLT%2CIBKR%2CFAQ%2CDCA%2CSNSXX%2CHYSA%2CKBW%2CRBC%2CCFRA%2CFDIC%2CLAC%2CUAW%2CRULES%2CHOUSE%2CSVB%2CRIP%2CTSLA%2CCVNA%2CUNH%2CUBS%2CMSOX%2COPEC%2CRKLB%2CNDA%2CAMZN%2CULA%2CCBDW%2CNFTG%2COLB%2CLRE%2CTLRY%2CPST%2CSIRI%2CVNQ%2CNET%2CWHAT%2CLTD%2CSCE%2CQQM%2CLOT%2CNEE%2CVIPSX%2CVMFXX%2CYTD%2CAPR%2CCIT%2CPER%2CATM%2CPBR%2CDMA%2CBOA%2CDOES%2CSOME%2CCDC%2CIHS%2CRIG&fields=regularMarketPreviousClose%2CfiftyDayAverage%2CregularMarketVolume%2CaverageDailyVolume3Month%2CregularMarketPrice%2CregularMarketChangePercent%2CfloatShares

> can you take a look on it? thanks in advance.

Yes to make sure I keep track of this would you mind making an issue on https://github.com/DylanAlloy/AutoDD_Rev2
